docs/api/openapi3.json

```diff
@@ -785,7 +785,7 @@
       "url": "https://www.elastic.co/licensing/elastic-license"
     },
     "title": "Infra API",
-    "version": "0.13.4"
+    "version": "0.13.5+dev"
```

Comment:
We still have this problem, where the main branch has the wrong version after a release, so any PRs have to fix this post release.